docker-compose.yaml

```diff
@@ -61,4 +73,6 @@ services:
             - '/etc/ryu/faucet:/etc/ryu/faucet'
         ports:
             - '6653:6653'
-            - '9244:9244'
+            - '9244'
+        links:
+            - db
```

Comment:
I propose we rename the "db" container to "influxdb" now that we have multiple databases in play.